### DISCUSSION TITLE
Unable to authorize

### DISCUSSION THREAD

**Discussion by Scholdan** [alexkn/node-red-contrib-homeconnect#8] [Q&A] [answered]
Hello 

I get this fault no matter what I'm doing:

error | "unauthorized_client"
-- | --
error_description | "client or redirection URI not authorized"

I got it working fine in home-assistant,

do I need to have 1880 open for the internet and no password on https://home-assistant.xxx.com1880/homeconnect/auth/callback

when I access https://home-assistant.xxx.com:1880/homeconnect/auth/callback I get a response of OK

**Comment by alexkn** [comment, marked as answer]
No, the callback url does not have to be accessible from the internet.
